opm/input/eclipse/Schedule/ScheduleState.hpp

```diff
@@ -60,6 +59,9 @@ namespace {
 
 namespace Opm {
 
+    namespace Action {
+        class Actions;
+    };
```

Comment:
I didn't pay enough attention to this whilst reviewing, but the trailing semicolon on the closing brace of the namespace declaration is superfluous and will generate warnings in some elevated warning settings.